src/cloudformation_cli_python_lib/metrics.py

```diff
@@ -119,11 +102,51 @@ def publish_log_delivery_exception_metric(
             "DimensionKeyExceptionType": str(type(error)),
             "DimensionKeyResourceType": self.resource_type,
         }
+        self.publish_metric(
+            metric_name=MetricTypes.HandlerException,
+            dimensions=dimensions,
+            unit=StandardUnit.Count,
+            value=1.0,
+            timestamp=timestamp,
+        )
+
+    @staticmethod
+    def _make_namespace(resource_type: str) -> str:
+        suffix = resource_type.replace("::", "/")
+        return f"{METRIC_NAMESPACE_ROOT}/{suffix}"
+
+
+class MetricsPublisherProxy:
```

Comment:
would u mind to leave a `__doc__` under this class, this would be helpful if we need later to make adjustments - this might be helpful for other devs

```python
    """[summary]

    Returns:
        [type]: [description]
    """"
```

Reply:
can do.